libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -379,6 +390,10 @@ class YulUtilFunctions
 	/// The resulting value might still have dirty higher order bits.
 	std::string prepareStoreFunction(Type const& _type);
 
+	/// @returns the name of a function that allocates stack.
+	/// signature: (size) -> memPtr
```

Comment:
We won't have stack addresses in Yul, only variables.
The stack is allocated and managed by LLVM.